### ISSUE TITLE
Is it possible to remove a Picker's underline?

### ISSUE THREAD

**Issue by toverux** [ChasakisD/Xamarin.Forms.BackgroundKit#61]
Hello,

I'm trying to wrap inputs in MaterialContentViews to given them elevation and rounded corners, I have an Entry and a Picker.

In Xamarin Forms Android, entries and pickers have an underline that cannot be removed via shared code.
Actually, I managed to remove it with Xamarin.Forms.BackgroundKit on entries, but not on the picker.

I tried various solutions without success, and here's the last iteration of it (clutter removed, but there's more code that should not be relevant to our problem):

```xaml
<bgKitControls:MaterialContentView 
    Background="{bgKitMarkup:BgProvider Elevation=4, Color=White}"
    IsCornerRadiusHalfHeight="True">

    <Picker bgKitEffects:BackgroundEffect.Background="{bgKitMarkup:BgProvider Color=White}" />
</bgKitControls:MaterialContentView>
```

Picker is the first control, Entry is the second:
![image](https://user-images.githubusercontent.com/1343250/74767025-5eee6b00-5286-11ea-9f24-138df60bc7c0.png)

(yeah the button has a terrible shadow, that's because I still use the one provided by Xamarin.Forms for now :p)

While the exact same technique works on entries, it does not here. Instead of, or accompanying, `Color=White` on the Picker, I also tried adding `BorderWidth=0` or `BorderColor=Transparent`.

I'm also still using `MaterialContentView` because of `IsCornerRadiusHalfHeight` and because I can apply a padding on it, which I can't on the Picker. I also seem to have less rendering problems (elevation shadow not rounded while the background, and more, but anyway).

Another test, without `MaterialContentView`, terrible rendering:

![image](https://user-images.githubusercontent.com/1343250/74767796-ceb12580-5287-11ea-8252-8c5ca2bb0c8e.png)

(This one has `{bgKitMarkup:BgProvider Color=White, CornerRadius=24, Elevation=4, BorderWidth=5, BorderColor=White}`) and some more (unrelated) Xamarin.Forms properties.

But maybe there's a better solution to my problem?

I'll also try on iOS tomorrow to see how that works on that platform.

A last word. Thank you so much for your work ; this must have been hard. I use Xamarin.Forms occasionally for years and this is probably the best library I've discovered for everyday work. I mean, really: if you've got a PayPal or something, that'd be my pleasure to make a little donation.

**Comment by toverux** [comment]
Update: so I tested on iOS and I get the usual border radius around the Picker and the Entry.
If that's expected behavior (then the underline suppression on the Entry on Android might be a bug), I will create custom renderers for them to remove their normal decoration.

**Comment by ChasakisD** [comment]
Hello @toverux,

Thank you very much for your feedback.

The border suppression is a feature and not a bug. I have a fix-ready on Android, but need some more time on iOS. You will be able to remove the default border by setting the Background of BackgroundEffect to null.

PS: If you would like to make a donation use this PayPal mail chasakisd[at]gmail[dot]com

**Comment by toverux** [comment]
Ok, awesome!

PS: check your inbox ;)

**Comment by ChasakisD** [comment]
@toverux Thank you very much! Just published version 2.0.4 to nuget.

In order to remove the border on picker use the following:
```xml
<Picker bgKitMarkup:BackgroundEffect.Background="{x:Null}" />
```

**Comment by toverux** [comment]
It's working, thank you very much!
